go/interface_spec.md

```diff
@@ -0,0 +1,11 @@
+# Interfaces
+
+When providing an interface for a package, it is convention to do this in a `spec.go` file. 
+
+Interfaces follow a naming convention of ending in _'er'_ to describe the behaviour. A `write` package, for example, should fulfil a `Writer` interface. A `scrape` package, should fulfil a `Scraper` interface.
```

Comment:
Except when you have something like `Collector`. 😛

Reply:
_/me opens dictionary_

I'll change the wording here, and include a `Collector` example 😅.

Reply:
Updated 👍